NewRepo.md

```diff
@@ -0,0 +1,28 @@
+Students are welcome to add new repositories to our Organization. Fill out this form https://goo.gl/forms/qJ2wXrD7UlKc0V8V2 to add your repository. 
+
+You should follow the guidelines stated below. Please read this carefully before you fill out the form.
+
+## Guidelines for maintainers
+1. The student should be enrolled in NIT Durgapur for the current Academic session when he adds the repository.
+2. Join our [gitter channel](https://gitter.im/NIT-dgp/General), [mailing list](https://groups.google.com/forum/?hl=en#!forum/nitdopensource) and [facebook group](https://www.facebook.com/groups/NITDgpOS).
+3. The student should have adequate involvement and contribution in our community.
```

Comment:
Yes they should have contributed actively on any of our repos. Well there is no certain benchmark. It is upto the Org Admins to decide their involvement. They can perhaps ask us before adding the repo if we are ready to accept his proposal.